networking/common/lib/lib_utils.sh

```diff
@@ -15,7 +15,7 @@ then
 	    echo $STANDALONE | grep -q $HOSTNAME
 	}
 	
-	rhts_submit_log()
+	rstrnt-report-log()
```

Comment:
Nice!

Although, I wonder, maybe it's better to drop the `rhts_`/`rstrnt-` prefix altogether, to avoid people thinking that it's a function defined elsewhere?

Reply:
rstrnt-report-log() is the replacement for legacy rhts_submit_log.  
legacy rhts has rhts- and rhts_ command, but  in  new restraint got replaced with rstrnt-
all rstrnt-xxxxx is  reserved for restraint command.

Reply:
Yeah, I meant that if someone sees a call to `rstrnt-report-log` somewhere, they will probably assume that the function is defined in restraint... but it's actually defined in `lib_utils.sh`.

Not a big deal, I just thought it might be slightly confusing for some people. Feel free to merge as-is.